baekjoon_9/1978.py

```diff
@@ -0,0 +1,24 @@
+# 1978 - 소수 찾기
+
+n = int(input())
+
+sosu = list(map(int, input().split()))  # 숫자열 입력
+
+result = 0
+
+for i in sosu:  # 입력된 문자열에서 하나씩 꺼내 쓰면...
+
+    cnt = 0  # 소수 판별용 카운터, 소수는 나누어지는 값이 자기 자신 하나여야 한다!
+
+    if i == 1:  # 1은 소수가 아니다
+        pass
+
+    for j in range(2, i + 1):  # 2부터~  sosu 안의 값 i 까지
```

Comment:
range 부분을 좀 더 수정하면 더 효율적으로 쓸 수 있어요.
왜 더 효율적인지는 스스로 생각해보고 답해주세요.
`math`라이브러리는 따로 추가해줘야 해요
```suggestion
    for j in range(2, int(math.sqrt(i)) + 1):
```